README.md

```diff
@@ -1,5 +1,7 @@
 # Pluto
 
+[![Build Status](https://dev.azure.com/samueltebbs/pluto/_apis/build/status/SamTebbs33.pluto?branchName=develop)](https://dev.azure.com/samueltebbs/samueltebbs/_build/latest?definitionId=1&branchName=develop)
+
```

Comment:
Also I don't know if this is possible, but could this also provide a link to the built binaries so the person can just download and run without having to faf with building them self's if they don't want to

Reply:
Good idea, will investigate. It may be good to make this a separate issue though.